installers/msi-language/StateDeploy/CustomAction.cs

```diff
@@ -398,6 +432,7 @@ public static ActionResult StateDeploy(Session session)
                 return ActionResult.Failure;
             }
 
+            TrackerSingleton.Instance.TrackEventSynchronously(session, "stage", "finished", "success");
```

Comment:
There's nothing else that can fail after this? Can't we track this a bit closer to the actual success state of the MSI installation?

Reply:
All "finished" commands are now triggered after the finishing MSI dialogs are displayed (CustomExitDialog => success, CustomFatalError => failure).